packages/autocomplete/src/elements/Multiselect.js

```diff
@@ -0,0 +1,449 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { MenuView, Item } from '@zendeskgarden/react-menus';
+import { TextGroup, Label, FauxInput, Input, Hint, Message } from '@zendeskgarden/react-textfields';
+import { Tag, Close } from '@zendeskgarden/react-tags';
+import { Anchor } from '@zendeskgarden/react-buttons';
+import { FieldContainer, KEY_CODES } from '@zendeskgarden/react-selection';
+import AutocompleteContainer from '../containers/AutocompleteContainer';
+
+const DEFAULT_NUM_TAGS = 4;
+const VALIDATION = {
+  SUCCESS: 'success',
+  WARNING: 'warning',
+  ERROR: 'error',
+  NONE: 'none'
+};
+
+const StyledNoItemsMessage = styled.div`
+  margin: 16px;
+  text-align: center;
+`;
+
+const StyledMenuOverflow = styled.div`
+  overflow-y: auto;
+  max-height: ${props => props.maxHeight || 'inherit'};
+`;
+
+const StyledInput = styled(Input)`
+  && {
+    flex-basis: 60px;
+    flex-grow: 1;
+    margin: 2px;
+  }
+
+  ${props =>
+    props.selectedValues.length !== 0 && // eslint-disable-line
+    (!props.isFocused || props.tagFocusedKey !== undefined) &&
+    !props.isOpen &&
+    !props.placeholder &&
+    `
+    &&& {
+      margin: 0;
+      opacity: 0;
+      height: 0;
+      min-height: 0;
+      width: 0;
+      min-width: 0;
+    }
+  `}
+`;
+
+const StyledFauxInput = styled(FauxInput)`
+  cursor: text;
+`;
+
+const StyledSpacedTag = styled(Tag)`
+  margin: 2px;
+`;
+
+const StyledMoreAnchor = styled(Anchor)`
+  && {
+    margin: 2px;
+  }
+`;
+
+export default class Multiselect extends Component {
+  static propTypes = {
+    label: PropTypes.string,
+    'aria-label': PropTypes.string,
+    validation: PropTypes.oneOf([
+      VALIDATION.SUCCESS,
+      VALIDATION.WARNING,
+      VALIDATION.ERROR,
+      VALIDATION.NONE
+    ]),
+    message: PropTypes.node,
+    hint: PropTypes.node,
+    selectedValues: PropTypes.arrayOf(PropTypes.string),
+    onChange: PropTypes.func,
+    inputRef: PropTypes.func,
+    placeholder: PropTypes.string,
+    maxHeight: PropTypes.string,
+    disabled: PropTypes.bool,
+    options: PropTypes.array.isRequired,
+    noOptionsMessage: PropTypes.string,
+    renderOption: PropTypes.func,
+    renderDropdown: PropTypes.func,
+    renderShowMore: PropTypes.func,
+    optionFilter: PropTypes.func,
+    closeOnSelect: PropTypes.bool
+  };
```

Comment:
Please add support for `small` and `focusInset`

Reply:
Forgot about these two 😓 Both are now supported